src/collections/_documentation/product/quotas.md

```diff
@@ -3,21 +3,27 @@ title: 'Quotas & Filtering'
 
-A subscription to Sentry primarily entails a resource quota on the number of events you can send within a window of time. These windows apply to a 60-second bucket for old plans and a one month bucket for new plans. Your subscription will provide a rate limit which is the maximum number of events the server will accept during that period on old plans, and the same can be configured for new plans. The server will respond with a 429 HTTP status code when this threshold has been reached.
-
-Additionally, some plans may also include a daily rate limit. For example, you’ll be restricted to a maximum of 250 events per day if you’re on the Limited plan.
+A subscription to Sentry primarily entails a resource quota on the number of events you can send within a month long period. The server will respond with a 429 HTTP status code when this threshold has been reached.
 
 ## Increasing Quotas
 
-Each tiered plan in Sentry has a predefined rate limit. The more expensive the plan, the more data you’re allowed to send. While the plans available will fit most individual and small business needs, there often arises a need for more. Fear not, Sentry is designed to handle large throughput, and if your team needs more, we’re happy to help. Reach out to your account manager, or send an email to [support@sentry.io](mailto:support%40sentry.io) to learn more about increasing capacity.
+Each tiered plan in Sentry has a predefined rate limit. The more expensive the plan, the more data you’re allowed to send. While the plans available will fit most individual and small business needs, there often arises a need for more. Fear not, Sentry is designed to handle large throughput, and if your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales%40sentry.io) to learn more about increasing capacity.
 
 ## Rate Limiting Projects {#id1}
 
-If you’re on our Large or Enterprise plan we support per-key rate limits. These allow you to set the maximum volume of events a key will accept during a period of time.
+Per-key rate limits allow you to set the maximum volume of events a key will accept during a period of time.
 
 For example, you may have a project in production that generates a lot of noise. With a rate limit you could set the maximum amount of data to “500 events per minute”. Additionally, you could create a second key for the same project for your staging environment which is unlimited, ensuring your QA process is still untouched.
 
-To setup rate limits, navigate to the Project you wish to limit, go to **[Project] » Client Keys**. Select on an individual key or create a new one, and you’ll be able to define a rate limit as well as see a breakdown of events received by that key.
+To setup rate limits, navigate to the Project you wish to limit, go to **[Project] » Client Keys**. Select an individual key or create a new one, and you’ll be able to define a rate limit as well as see a breakdown of events received by that key.
```

Comment:
It might be worth adding **[Project] » Client Keys » Details** here since you have to expand to see the setting.